lib/core_types/list.dart

```diff
@@ -8,3 +8,12 @@ AST visitListProperties(AST node, AST left) {
       return intAST;
   }
 }
+
+AST visitListMethods(AST node, AST left) {
+  switch (node.binaryOpRight.funcCallExpression.variableName) {
```

Comment:
Since this is still not implemented it should just throw. Leave the todo, but be more descriptive of what needs to be done